client/src/components/pages/CreateProgram/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import axios from 'axios';
+import Header from "../../Header";
+import MainForm from '../../Form'
+import {Title, StyledPage} from "./index.style";
+
+class CreateProgram extends Component {
+
+  getFieldList() {
+    const fields = [
```

Comment:
Pun this on the render function before the return.

Reply:
Done :+1: